.github/workflows/helm-publish.yaml

```diff
@@ -0,0 +1,38 @@
+name: Publish Helm Charts
+
+on:
+  push:
+
+jobs:
+  get-version:
+    name: Get version from bumpversion
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get-version.outputs.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Get current version from bumpversion
+        id: get-version
+        run: |
+          version=$(cat .bumpversion.cfg | sed -nE 's/.*current_version = (.*)/\1/p')
+          if [[ "${GITHUB_REF#refs/heads/}" != "main" && "${GITHUB_REF#refs/heads/}" != "master" ]]; then
+            version="${version}-SNAPSHOT"
+          fi
```

Comment:
None snapshot versions should only ever be build and published on Tag branches, not on default branches, so that after merging changes to main the manual release still needs to be triggered in CI. Also having a suffix should be possible through configuration of the .bumpversion.cfg file directly.

Reply:
Do we even use -SNAPSHOT for bumpversion?

Reply:
This was meant to be similar to what is in the current streams-bootstrap repo. I'll look into bumpversion to configure it proper

Reply:
Will be addressed in a later PR